locales/fr/yourloops.json

```diff
@@ -369,6 +369,8 @@
   "today": "Aujourd'hui",
   "tooltip-release-notes": "Notes de version",
   "training": "Formation",
+  "training-body": "Nouveau <trainingLink>{{training}}</trainingLink> disponible, veuillez le lire avant de continuer sur yourloops. Vous pouvez également le lire plus trad, en cliquant sur le lien disponible en bas de page.",
```

Comment:
We usually don't translate other languages than english. It's not big deal though